tools/index.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: default
+---
+## Advanced Tools
+
+WebAssembly is supported by a number of tools to help developers build and process source files and generated binary content. If you're a compiler writer, enjoy low level coding or just want to experiment with the raw WebAssembly format, these tools are for you.
+
+<div class="flash flash-warn flash-messages">
+Note, at the date of writing - Oct 2016, the WebAssembly binary format is close to final (expected to be stable) and developer feedback is encouraged before the format is locked down for version 1.0 which is expected to be early 2017. Browser previews of WebAssembly support can be used for experimentation and feedback can be sent to the <a href="https://www.w3.org/community/webassembly/">community group</a> or by posting issues on the <a href="https://github.com/WebAssembly/design">design repository</a>.
+</div>
+
+There are two distinct sets of tools that are of interest to compiler writers or developers who want to work with WebAssembly binary generated by other tools like [Emscripten](http://kripken.github.io/emscripten-site/):
```

Comment:
"there are currently"? I'd rather not make it sound like people can't do their own thing if they want to.